server/app.js

```diff
@@ -107,6 +109,10 @@ async function main() {
       next();
     }, mongooseUtilRouter);
 
+    app.use('/request/', (req, res, next) => {
```

Comment:
Since we are not adding middleware here, this can just be `app.use('/request/'. requestRouter)`